crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -448,6 +448,11 @@ define_tedge_config! {
             /// Enable auto registration feature
             #[tedge_config(example = "true", default(value = true))]
             auto_register: bool,
+
+            /// The directory where the entity store is persisted across restarts
+            #[tedge_config(example = "/data/tedge", default(value = "/data/tedge"))]
+            #[doku(as = "PathBuf")]
+            path: Utf8PathBuf,
```

Comment:
> Beware that tedge-agent doesn't require this directory to exist and use /etc/tedge/.agent as a fallback. Do you plan something similar for the mapper?

Yes, a default is required. In fact all the test failures were due to that. 

> It might be better to consider a single tedge.db.path for all these use cases where things must be persisted across restarts.

Agree